src/object/player.cpp

```diff
@@ -761,6 +762,18 @@ Player::update(float dt_sec)
     }
   }
 
+  // break bricks above without stopping
+  if (m_stone && m_physic.get_velocity_y() < 30.f)
+  {
+    Rectf topbox = get_bbox().grown(-1.f);
+    topbox.set_top(get_bbox().get_top() - 16.f);
+    for (auto& brick : Sector::get().get_objects_by_type<Brick>()) {
+      if (topbox.contains(brick.get_bbox())) {
```

Comment:
You can either follow the convention:

```
if (...)
{
}
else
{
}
```

Or even better for one-line statements, remove the curly brackets as a whole.
Applies to all changes.